radish34/messenger/Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM node:12-alpine
+FROM radish34_logger as logger
+FROM node:12.16
 
-# Create app directory
-WORKDIR /app
+ENV FORCE_COLOR=1
+
+RUN mkdir /logger
+WORKDIR /logger
```

Comment:
It seem like there are some redundant statements in this docker file. Do you need to do the two above statements in this file since they're already part what you do in the base image `radish34_logger`?

Reply:
Indeed, those statements look kind of redundant. But I would prefer these statements instead of copying everything (including `src` files and `node_modules`) from the `radish34_logger` image with `COPY --from=logger /logger/ /logger/`.